SvrfSDK/Source/SvrfSDK.swift

```
@@ -220,7 +292,14 @@ public class SvrfSDK: NSObject {
         return faceFilter
     }
     
-    //MARK: private functions
+    // MARK: private functions
+    /**
+     Renders a *SCNScene* from a *Media*'s glb file using *SvrfGLTFSceneKit*.
+
+     - parameters:
+        - media: The *Media* to return a *SCNScene* from.
+     - returns: SCNScene?
+     */
     private static func getSceneFromMedia(media: Media) -> SCNScene? {
```

This should also return an error if there's a problem generating the `SCNNode`

---

Yea, I'll update readme after updating the project